Extension/src/LanguageServer/configurations.ts

```diff
@@ -759,13 +758,14 @@ export class CppProperties {
             if (recursive) {
                 res = res.slice(0, res.length - 2);
             }
+            // fastGlob can't deal with backslash-separated path => remove them
+            const normalized = res.replace(/\\/g,'/');
+            const cwd = this.rootUri?.fsPath?.replace(/\\/g,'/');
             // fastGlob silently strip non-found paths. limit that behavior to dynamic paths only
-            if (fastGlob.isDynamicPattern(res)) {
-                globResult.concat(fastGlob.sync(res, { onlyDirectories: true, globstar: false, cwd: this.rootUri?.fsPath })
-                    .map(s => recursive ? s + '**' : s));
-            } else {
-                globResult.push(recursive ? res + '**' : res);
-            }
+            const matches = fastGlob.isDynamicPattern(normalized) ?
+                            fastGlob.sync(normalized, { onlyDirectories: true, /*globstar: false,*/ cwd}):
```

Comment:
In my case I had to use a **middle** recursive-glob `path/**/headers`
But note that **ending** recursive glob are processed like before (not by fastglob)